### DISCUSSION TITLE
Morse Runner

### DISCUSSION THREAD

**Discussion by nealey** [Vail-CW/vail_repeater_depricated#60] [Ideas]
A Vail user who may prefer to be anonymous (or may not: I'm asking now!) has coded up a sort of "practice bot" that incorporates a bunch of ham radio isms which would probably benefit a lot of users. They have a prototype that includes a couple things people have been asking for with Vail, such as noise and different tone frequencies.

I'm starting this discussion in the hopes of luring them out into the open so we can talk about possibly merging some of their work into Vail!

**Comment by mgiugliano** [comment]
As I moved ..._faster than the speed of light_, for a recent work/life relocation, some time warping happened and I coming back almost 3 years later. I have a renew interest in having vail clients as well as (maybe) BOT vail clients, along the lines of Morse Runner. That was a software to interact with a sort of BOT too! I am here to chat and discuss it with you all.
